team.html

```diff
@@ -1,333 +1,472 @@
+            <li><strong>Ignasi Reichardt</strong> (<a href="https://github.com/ignasi-reichardt>">@ignasi-reichardt</a>), APC,
+                France</li>
+            <li><strong>Johannes King</strong> (<a href="https://github.com/joleroi">@joleroi</a>), MPIK, Germany</li>
+            <li><strong>Jonathan Harris</strong> (<a href="https://github.com/JonathanDHarris">@JonathanDHarris</a>), JON, UK
+            </li>
+            <li><strong>José Enrique Ruiz</strong> (<a href="https://github.com/Bultako">@Bultako</a>), IAA, Spain</li>
+            <li><strong>Julien Lefaucheur</strong> (<a href="https://github.com/jjlk">@jjlk</a>), SAP/CEA, France</li>
+            <li><strong>Lars Mohrmann</strong> (<a href="https://github.com/lmohrmann">@lmohrmann</a>), Erlangen University, Germany
+            </li>
+            <li><strong>Léa Jouvin</strong> (<a href="https://github.com/JouvinLea">@JouvinLea</a>), IFAE, Spain</li>
+            <li><strong>Luigi Tibaldo</strong> (<a href="https://github.com/tibaldo">@tibaldo</a>), IRAP, France</li>
+            <li><strong>Manuel Paz Arribas</strong> (<a href="https://github.com/mapazarr">@mapazarr</a>), Humboldt University,
+                Germany</li>
+            <li><strong>Matthew Wood</strong> (<a href="https://github.com/woodmd">@woodmd</a>), SLAC, US</li>
+            <li><strong>Matthias Wegen</strong> (<a href="https://github.com/wegenmat">@wegenmat</a>), DESY Zeuthen, Germany</li>
+            <li><strong>Nachiketa Chakraborty</strong> (<a href="https://github.com/cnachi">@cnachi</a>), MPIK, Germany</li>
+            <li><strong>Olga Vorokh</strong> (<a href="https://github.com/OlgaVorokh">@OlgaVorokh</a>),, Belarus</li>
+            <li><strong>Peter Deiml</strong> (<a href="https://github.com/pdeiml">@pdeiml</a>), Erlangen Univ, Germany</li>
+            <li><strong>Régis Terrier</strong> (<a href="https://github.com/registerrier">@registerrier</a>), APC, France</li>
+            <li><strong>Roberta Zanin</strong> (<a href="https://github.com/robertazanin">@robertazanin</a>), MPIK, Germany</li>
+            <li><strong>Rolf Bühler</strong> (<a href="https://github.com/rbuehler">@rbuehler</a>), DESY Zeuthen, Germany</li>
+            <li><strong>Rubén López-Coto</strong> (<a href="https://github.com/rlopezcoto">@rlopezcoto</a>), Padova Univ, Italy
+            </li>
+            <li><strong>Stefan Klepser</strong> (<a href="https://github.com/klepser">@klepser</a>), DESY Zeuthen, Germany</li>
+            <li><strong>Thomas Armstrong</strong> (<a href="https://github.com/thomasarmstrong">@thomasarmstrong</a>), Manchester
+                University, UK
+            </li>
+            <li><strong>Thomas Vuillaume</strong> (<a href="https://github.com/vuillaut">@vuillaut</a>), LAPP, France</li>
+            <li><strong>Victor Zabalza</strong> (<a href="https://github.com/zblz">@zblz</a>), Leads Univ, UK<sup>*</sup></li>
```

Comment:
You left the marker here.

Reply:
The description of the marker is missing, non? * means 'Left the project'.
It is important to keep on this list people that left because 1/ they just do contribute, 2/ it gives an 'official' recognition for their work